src/apps/ENGINE/src/compiler.cpp

```diff
@@ -6593,7 +6613,7 @@ bool COMPILER::LoadState(std::fstream &fileS)
         pString = ReadString();
         if (pString)
         {
-            Assert(utf8::IsValidUtf8(pString));
+            //Assert(utf8::IsValidUtf8(pString));
```

Comment:
Should we log this behaviour? It may help detecting non-converted files

Reply:
since now there is a check in the `ReadString` function, no need to have the same check here. I removed it.